tools/c7n_azure/tests/test_storage.py

```diff
@@ -19,6 +19,24 @@
 class StorageTest(BaseTest):
     def setUp(self):
         super(StorageTest, self).setUp()
+        cleanup_policy = self.load_policy({
```

Comment:
generally we'd prefer to have a cleanup like this be a client api code, else we're depending on the thing we're testing for the tests to run correctly.